### PR TITLE
Apidoc

### PR DESCRIPTION
**Pull request by ankitpws**
Added API Doc

### CODE REVIEW

File: src/api-doc/api-doc.yaml [hunk 1]
```diff
@@ -0,0 +1,2869 @@
+---
+openapi: 3.0.0
+info:
+  title: Elevate Mentoring
+  version: 1.0.0
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+  description: >-
+    - The Mentoring Service is a centralized Service to support other services. Apis perform operations related to user entities notification etc
```
Comment (rakeshSgr) [inline src/api-doc/api-doc.yaml:8]: @ankitpws please update the valid description

Comment (rakeshSgr) [reply src/api-doc/api-doc.yaml:8]: remove notification

File: src/api-doc/api-doc.yaml [hunk 2]
```diff
@@ -0,0 +1,2869 @@
...
+  title: Elevate Mentoring
+  version: 1.0.0
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+  description: >-
+    - The Mentoring Service is a centralized Service to support other services. Apis perform operations related to user entities notification etc 
+
+    - The URL for Mentoring API(s) is `{context}/mentoring/v1`
+    - <b>Note:</b> These resources can be used in other services
+  contact:
+    email: info@sunbird.org
```
Comment (rakeshSgr) [inline src/api-doc/api-doc.yaml:13]: @ankitpws remove this email

Comment (rakeshSgr) [reply src/api-doc/api-doc.yaml:13]: use this tech-infra@shikshalokam.org

File: src/api-doc/api-doc.yaml [hunk 3]
```diff
@@ -0,0 +1,2869 @@
+---
+openapi: 3.0.0
+info:
+  title: Elevate Mentoring
+  version: 1.0.0
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
```
Comment (rakeshSgr) [inline src/api-doc/api-doc.yaml:6]: @ankitpws  this we need to remove  in all the api docs

File: src/api-doc/api-doc.yaml [hunk 4]
```diff
@@ -0,0 +1,2869 @@
...
+                '$ref': '#/components/schemas/sessions/startSession400Response'
+        '500':
+          description: Bad Request. Could not create meeting.
+          content:
+            application.json:
+              schema:
+                '$ref': '#/components/schemas/sessions/startSession500Response'
+  '/mentoring/v1/sessions/update':
+    post:
+      summary: Create session
```
Comment (rakeshSgr) [inline src/api-doc/api-doc.yaml:538]: @ankitpws  session /create or update

Comment (rakeshSgr) [inline src/api-doc/api-doc.yaml:538]: @ankitpws session create/update

File: src/api-doc/api-doc.yaml [hunk 5]
```diff
@@ -0,0 +1,2869 @@
...
+          content:
+            application.json:
+              schema:
+                '$ref': '#/components/schemas/questionset/readQuestionSetResponse400'
+  '/mentoring/v1/feedback/forms/{SessionId}':
+    get:
+      summary: Get feedback form
+      tags:
+        - Feedback Form APIs
+      description: "This API is associated with getting feedback form in mentoring \n - Then
```
Comment (rakeshSgr) [inline src/api-doc/api-doc.yaml:1045]: @ankitpws feedback form  in mentoring => feedback forms of the session.

File: src/api-doc/api-doc.yaml [hunk 6]
```diff
@@ -0,0 +1,2869 @@
...
+          content:
+            application.json:
+              schema:
+                '$ref': '#/components/schemas/feedback/getFeedbackFormResponse400'
+  '/mentoring/v1/feedback/submit/{SessionId}':
+    post:
+      summary: Submit feedback
+      tags:
+        - Feedback Form APIs
+      description: "This API is associated with submit feedback form in mentoring \n - Then
```
Comment (rakeshSgr) [inline src/api-doc/api-doc.yaml:1082]: @ankitpws  not require ( in mentoring ) as its a mentoring api doc

Comment (rakeshSgr) [reply src/api-doc/api-doc.yaml:1082]: check other apis also

File: src/api-doc/api-doc.yaml (hunk 1 above)
Comment (ankitpws) [reply src/api-doc/api-doc.yaml]: fixed

File: src/api-doc/api-doc.yaml (hunk 2 above)
Comment (ankitpws) [reply src/api-doc/api-doc.yaml]: fixed

File: src/api-doc/api-doc.yaml (hunk 6 above)
Comment (ankitpws) [reply src/api-doc/api-doc.yaml]: fixed

File: src/api-doc/api-doc.yaml (hunk 3 above)
Comment (ankitpws) [reply src/api-doc/api-doc.yaml]: fixed

**Pull request merged**